\hypertarget{classaiohttp_1_1client__exceptions_1_1_client_proxy_connection_error}{}\section{aiohttp.\+client\+\_\+exceptions.\+Client\+Proxy\+Connection\+Error Class Reference}
\label{classaiohttp_1_1client__exceptions_1_1_client_proxy_connection_error}\index{aiohttp.\+client\+\_\+exceptions.\+Client\+Proxy\+Connection\+Error@{aiohttp.\+client\+\_\+exceptions.\+Client\+Proxy\+Connection\+Error}}


Inherits \hyperlink{classaiohttp_1_1client__exceptions_1_1_client_connector_error}{aiohttp.\+client\+\_\+exceptions.\+Client\+Connector\+Error}.

\subsection*{Additional Inherited Members}


\subsection{Detailed Description}
\begin{DoxyVerb}Proxy connection error.

Raised in :class:`aiohttp.connector.TCPConnector` if
    connection to proxy can not be established.
\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/lib/python3.\+6/site-\/packages/aiohttp/client\+\_\+exceptions.\+py\end{DoxyCompactItemize}
